packages/x-components/src/x-modules/scroll/components/main-scroll-item.vue

```diff
@@ -105,14 +73,12 @@
         oldObserver: ScrollVisibilityObserver | null
       ): void => {
         {
-          if (rootRef.value !== null) {
-            const htmlElement = isElementRef(rootRef.value) ? rootRef.value.$el : rootRef.value;
```

Comment:
If someone can say/demonstrate me why is it necessary, recojo cable

Reply:
I'm pretty sure that we had an issue that led to this code but I can't remember the specifics. I've trying to replicate it and failed. Maybe it was a side effect due to other thing that we already changed, I don't know 🤷‍♂️ Anyway, seems that this is working just fine now